src/components/form/Text.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { TextInput } from 'hds-react';
+
+import type OptionType from '../../types/OptionType';
+
+type TextProps = {
+  componentId: string;
+  initialValue: string[];
+  initialize: Function;
+  label: string;
+  placeholder: string;
+  setQuery: Function;
+  searchState: any;
```

Comment:
There's a type for searchState in the news app, which might be somewhat copy-able here to get rid of `any`